src/Microsoft.Health.SqlServer/Features/Schema/SchemaUpgradeRunner.cs

```diff
@@ -19,88 +19,88 @@ public class SchemaUpgradeRunner
     {
         private readonly IScriptProvider _scriptProvider;
         private readonly IBaseScriptProvider _baseScriptProvider;
-        private readonly SqlServerDataStoreConfiguration _sqlServerDataStoreConfiguration;
         private readonly IMediator _mediator;
         private readonly ILogger<SchemaUpgradeRunner> _logger;
+        private readonly ISqlConnectionFactory _sqlConnectionFactory;
 
         public SchemaUpgradeRunner(
             IScriptProvider scriptProvider,
             IBaseScriptProvider baseScriptProvider,
-            SqlServerDataStoreConfiguration sqlServerDataStoreConfiguration,
             IMediator mediator,
-            ILogger<SchemaUpgradeRunner> logger)
+            ILogger<SchemaUpgradeRunner> logger,
+            ISqlConnectionFactory sqlConnectionFactory)
         {
             EnsureArg.IsNotNull(scriptProvider, nameof(scriptProvider));
             EnsureArg.IsNotNull(baseScriptProvider, nameof(baseScriptProvider));
-            EnsureArg.IsNotNull(sqlServerDataStoreConfiguration, nameof(sqlServerDataStoreConfiguration));
             EnsureArg.IsNotNull(mediator, nameof(mediator));
             EnsureArg.IsNotNull(logger, nameof(logger));
+            EnsureArg.IsNotNull(sqlConnectionFactory, nameof(sqlConnectionFactory));
 
             _scriptProvider = scriptProvider;
             _baseScriptProvider = baseScriptProvider;
-            _sqlServerDataStoreConfiguration = sqlServerDataStoreConfiguration;
             _mediator = mediator;
             _logger = logger;
+            _sqlConnectionFactory = sqlConnectionFactory;
         }
 
-        public void ApplySchema(int version, bool applyFullSchemaSnapshot)
+        public async Task ApplySchema(int version, bool applyFullSchemaSnapshot)
```

Comment:
Keep the naming consistent. ApplySchemaAsync and ApplyBaseSchemaAsync

Reply:
A fix for this is made in [AB#106](https://github.com/microsoft/healthcare-shared-components/pull/106).